docs/app/Examples/views/Statistic/Types/Groups.js

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import { Statistic } from 'stardust'
 
 const { Group, Label, Value } = Statistic
+
 const items = [
-  { label: 'Faves', value: 22 },
+  { label: 'Faves', value: '22' },
   { label: 'Views', value: '31,200' },
-  { label: 'Members', value: 22 },
+  { label: 'Members', value: '22' },
```

Comment:
I think we should just change the propType validation to string/number.  Since technically it _could_ support `node` as it ends up here:

```jsx
<div {...rest} className={classes}>{children || value}</div>
```